src/custom-elements/src/components/helper-message/helper-message.scss

```diff
@@ -0,0 +1,4 @@
+:host(chi-label) {
```

Comment:
@pedrotaglialatela-lumen if the component is helper-message, for what is needed chi-label?